src/Response.php

```diff
@@ -204,6 +204,15 @@ public function writeHead($status = 200, array $headers = array())
             );
         }
 
+        // assign date header if no 'date' is given, use the current time where this code is running
+        if (!isset($lower['Date'])) {
+            // example: "Date: Tue, 15 Nov 1994 08:12:31 GMT"
+            $headers = array_merge(
+                array('Date' => date('D, d M Y H:i:s T')),
```

Comment:
Thanks for the link!

> When a sender generates a
   header field that contains one or more timestamps defined as
   HTTP-date, the sender MUST generate those timestamps in the
   IMF-fixdate format.
   […]
 > Preferred format:
 >     IMF-fixdate  = day-name "," SP date1 SP time-of-day SP GMT